src/models/devicemodel.cpp

```diff
@@ -12,7 +12,6 @@
 #include <QJsonDocument>
 #include <QJsonObject>
 #include <QJsonValue>
-#include <QMutableListIterator>
```

Comment:
Did this sneak in by mistake or should this line stay?

Reply:
Remove it :) Good catch!